CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ This project attempts to follow [semantic versioning](https://semver.org/)
 
 * _nada_
 
+## 1.0.1 - 2018-08-15
+
+* bug fixes
+  * Update alienvault role to handle Ubuntu and Amazon ansible\_distributions.
```

Comment:
is that `\` necessary?

Reply:
I think as part of the normal markdown spec it is. Github-flavored markdown might not care if it's escaped or not. Not really sure.